.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,145 @@
+# Contributing
+
+Thanks for your interest in contributing to vocs! Please take a moment to review this document before submitting a pull request.
+
+If you want to contribute, but aren't sure where to start, you can create a [new discussion](https://github.com/wevm/vocs/discussions).
+
+> Note
+> 
+> Please ask first before starting work on any significant new features.
+> 
+> It's never a fun experience to have your pull request declined after investing time and effort into a new feature. To avoid this from happening, we request that contributors create a [feature request](https://github.com/wevm/vocs/discussions/new?category=ideas) to first discuss any API changes or significant new ideas.
+
+## Basic Guide
+
+This guide is intended to help you get started with contributing. By following these steps, you will understand the development process and workflow.
+
+1. Cloning the repository
+2. Installing Node.js and Bun
+3. Installing dependencies
+4. Writing documentation
+5. Submitting a pull request
+6. Versioning
+
+---
+
+### Cloning the repository
+
+To start contributing to the project, clone it to your local machine using git:
+
+```bash
+git clone https://github.com/wevm/vocs.git
+```
+
+Or the [GitHub CLI](https://cli.github.com):
+
+```bash
+gh repo clone wevm/vocs
+```
+
+<div align="right">
+  <a href="#basic-guide">&uarr; back to top</a></b>
+</div>
+
+---
+
+### Installing Node.js and Bun
```

Comment:
I think we can remove references to Bun as we use pnpm.

Reply:
Done